querybook/webapp/components/DataDocSchedule/DataDocScheduleForm.tsx

```diff
@@ -215,6 +219,14 @@ export const DataDocScheduleForm: React.FunctionComponent<
                 isValid,
                 dirty,
             }) => {
+                const disableIfRunningDocField = (
+                    <SimpleField
+                        label="Prevent Overlapping Runs"
```

Comment:
let's use the positive way to state this as `Allow Concurrent Runs`, make the default as allow to match the current behavior